src/DRICH_geo.cpp

```diff
@@ -546,29 +574,94 @@ static Ref_t createDetector(Detector& desc, xml::Handle_t handle, SensitiveDetec
               Translation3D(sensorSphRadius, 0., 0.) * // push radially to spherical surface
               RotationY(M_PI / 2) *                    // rotate sensor to be compatible with generator coords
               RotationZ(-M_PI / 2);                    // correction for readout segmentation mapping
-          auto sensorPV = gasvolVol.placeVolume(sensorVol, sensorPlacement);
+	    auto sensorPV = gasvolVol.placeVolume(sensorVol, sensorPlacement);
```

Comment:
Avoid using tabs (and set your editor to insert spaces instead). Tab width is not necessarily the same on every system.